### PR TITLE
Add voltage init ratio tap transformers modifications

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/entities/equipment/modification/VoltageInitTransformerModificationEmbeddable.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
```
Comment (ghazwarhili) [inline src/main/java/org/gridsuite/modification/server/entities/equipment/modification/VoltageInitTransformerModificationEmbeddable.java:2]: 20**23**

Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/modification/server/entities/equipment/modification/VoltageInitTransformerModificationEmbeddable.java]: Fixed

File: src/main/java/org/gridsuite/modification/server/modifications/VoltageInitModification.java [hunk 2]
```diff
@@ -62,5 +75,79 @@ public void apply(Network network, Reporter subReporter) {
...
+                    Reporter genReporter = subReporter.createSubReporter(THREE_WINDINGS_TRANSFORMER_MSG + t.getTransformerId(), THREE_WINDINGS_TRANSFORMER_MSG + t.getTransformerId());
+                    genReporter.report(Report.builder()
+                        .withKey("3WindingsTransformerNotFound")
+                        .withDefaultMessage("3 windings transformer with id=${id} not found")
+                        .withValue("id", t.getTransformerId())
+                        .withSeverity(TypedValue.WARN_SEVERITY)
+                        .build());
+                    return;
+                }
+                if (threeWindingsTransformer.getLeg(t.getLegSide()).getRatioTapChanger() == null) {
```
Comment (dbraquart) [inline src/main/java/org/gridsuite/modification/server/modifications/VoltageInitModification.java:97]: I wonder if we could have a NPE if getLeg() returns null.

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: request change
tests: done, but I can't see any transformer in the resulting modification. dont know why...

File: src/main/java/org/gridsuite/modification/server/modifications/VoltageInitModification.java (hunk 2 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/modification/server/modifications/VoltageInitModification.java:97]: It's impossible : a three windings transformer has necessarly three legs ...

File: src/main/java/org/gridsuite/modification/server/dto/VoltageInitTransformerModificationInfos.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Schema(description = "Voltage init transformer modfication infos")
```
[the remark this answers is not in the document]
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/modification/server/dto/VoltageInitTransformerModificationInfos.java:25]: Fixed

**Review by dbraquart — APPROVED** [review]
code: ok
tests: done in Front PR

**Pull request merged**